core/src/main/java/org/apache/brooklyn/core/mgmt/internal/LocalManagementContext.java

```diff
@@ -224,6 +224,7 @@ public String getManagementPlaneId() {
     public void setManagementPlaneId(String newPlaneId) {
         if (managementPlaneId != null && !managementPlaneId.equals(newPlaneId)) {
             log.warn("Management plane ID changed from {} to {}", managementPlaneId, newPlaneId);
+            log.debug("Management plane ID changed from {} to {}", new Object[] {managementPlaneId, newPlaneId}, new RuntimeException("Stack trace for setManagementPlaneId"));
```

Comment:
I think this actually gives you `debug(String, Object, Object)` rather than an overload that knows about `Throwable`, so you'll not get the right parameter matching here.